recipes/wavelet_buffer/all/test_package/conanfile.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from conan import ConanFile
+from conan.tools.build import cross_building
+from conan.tools.cmake import CMake
+from conan.tools.layout import cmake_layout
+
+
+class HelloTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    # VirtualBuildEnv and VirtualRunEnv can be avoided if "tools.env.virtualenv:auto_use" is defined
+    # (it will be defined in Conan 2.0)
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualBuildEnv", "VirtualRunEnv"
+    apply_env = False
```

Comment:
```suggestion
    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
    test_type = "explicit"
```
- Use test_type always: https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#changes-in-the-test-package-recipe
- VirtualBuildEnv is not needed here, because there is no build requirement